astropy/modeling/functional_models.py

```diff
@@ -349,6 +354,17 @@ def __init__(self, amplitude=amplitude.default, x_mean=x_mean.default,
                 y_vec = eig_vecs[:, 0]
                 theta = np.arctan2(y_vec[1], y_vec[0])
 
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        # stddev must be non-zero and positive.
+        if 'bounds' not in kwargs:
+            kwargs['bounds'] = {'x_stddev': (FLOAT_EPSILON, None),
+                                'y_stddev': (FLOAT_EPSILON, None)}
+        else:
+            kwargs['bounds']['x_stddev'] = kwargs['bounds'].get(
+                'x_stddev', (FLOAT_EPSILON, None))
+            kwargs['bounds']['y_stddev'] = kwargs['bounds'].get(
+                'y_stddev', (FLOAT_EPSILON, None))
+
```

Comment:
Any idea why simply specifying the bounds in Parameter doesn't work? Does it just get ignored? I think we should get to the bottom of this and either fix it or add a comment to explain why we can't do this.